data_juicer/utils/unittest_utils.py

```diff
@@ -2,11 +2,30 @@
 import shutil
 import unittest
 
+import numpy
+import pyarrow as pa
+import ray.data as rd
+from datasets import Dataset
+
+from data_juicer.ops import Filter
+from data_juicer.utils.constant import Fields
 from data_juicer.utils.registry import Registry
 
 SKIPPED_TESTS = Registry('SkippedTests')
 
 
+def TEST_TAG(*tags):
```

Comment:
There are two type of tags, while the PR's description mentioned 5 types of test scenarios. So what is the mapping relationship? 
```
single process
multi process
multi process with GPU
ray with CPU
ray with GPU
```

Reply:
For now, there is no distinction between CPU and GPU testing, and the corresponding configuration will be added in the future when needed.